common/src/calendar/__tests__/calendar.test.ts

```diff
@@ -781,9 +828,9 @@ describe("isReservationUnitReservable", () => {
     openingHours: {
       openingTimes: [
         {
-          date: new Date().toISOString(),
-          startTime: `${new Date().toISOString()}T04:00:00+00:00`,
-          endTime: `${new Date().toISOString()}T20:00:00+00:00`,
+          date,
+          startTime: `${date}T04:00:00+00:00`,
+          endTime: `${date}T20:00:00+00:00`,
```

Comment:
How does this work? how did it even work before?
`${new Date().toISOString()}T20:00:00+00:00` ~= '2023-02-23T07:45:05.193ZT20:00:00+00:00'
So it has double T... strings.

Reply:
By chance I guess. Corrected.